clients/hadoopfs/examples/spark_with_lakefs.py

```diff
@@ -0,0 +1,27 @@
+from pyspark.sql import SparkSession
+import pyspark
+import sys
+
+spark = SparkSession.builder.appName("test_app").getOrCreate()
+spark._jsc.hadoopConfiguration().set("fs.lakefs.access.key", "AKIAIOSFODNN7EXAMPLE")
+spark._jsc.hadoopConfiguration().set("fs.lakefs.secret.key", "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY")
```

Comment:
did you mean to keep this secret here?

Reply:
yes, this is the sample key/secret used in AWS documentation as an example